src/python/pants/backend/jvm/tasks/coverage/cobertura.py

```diff
@@ -140,15 +141,43 @@ def initialize_instrument_classpath(settings, targets, instrumentation_classpath
         settings.log.debug(
           "runtime_classpath ({}) cloned to instrument_classpath ({})".format(path, new_path))
 
-  def instrument(self):
+  def _iter_datafiles(self, output_dir):
+    for root, _, files in safe_walk(output_dir):
+      for f in files:
+        if f == self._DATAFILE_NAME:
+          yield os.path.join(root, f)
+          break
+
+  @property
+  def canonical_datafile(self):
+    if self._canonical_datafile is None:
```

Comment:
Does this need to be initialized in the constructor to avoid an undeclared property type error?

Reply:
Good call, fixed.